operator/gitops/argocd/tekton-results/configmap.yaml

```
@@ -10,3 +10,32 @@ metadata:
     app.kubernetes.io/version: v0.4.0
   name: tekton-results-database
   namespace: tekton-results
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: tekton-results-config
+  namespace: tekton-results
+data:
+  config: |
+    DB_USER=
+    DB_PASSWORD=
+    DB_HOST=
+    DB_PORT=5432
+    DB_NAME=
+    DB_SSLMODE=disable
+    GRPC_PORT=50051
+    REST_PORT=8080
+    PROMETHEUS_PORT=9090
+    TLS_HOSTNAME_OVERRIDE=
+    TLS_PATH=/etc/tls
+    LOG_TYPE=File
+    LOG_CHUNK_SIZE=32768
+    LOGS_DATA=
+    S3_BUCKET_NAME=
+    S3_ENDPOINT=
+    S3_HOSTNAME_IMMUTABLE=false
+    S3_REGION=
+    S3_ACCESS_KEY_ID=
+    S3_SECRET_ACCESS_KEY=
+    S3_MULTI_PART_SIZE=5242880
```

```suggestion
    S3_MULTI_PART_SIZE=5242880
    
```
Terminating newline.

---

Will push an updated verison